packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -774,18 +774,40 @@ export const Webchat = forwardRef((props, ref) => {
   }
 
   const userInputEnabled = isUserInputEnabled()
-  const emojiPickerEnabled = getThemeProperty(
-    WEBCHAT.CUSTOM_PROPERTIES.enableEmojiPicker,
-    props.enableEmojiPicker
-  )
-  const attachmentsEnabled = getThemeProperty(
-    WEBCHAT.CUSTOM_PROPERTIES.enableAttachments,
-    props.enableAttachments
-  )
+
+  const isEmojiPickerEnabled = () => {
+    const hasCustomEmojiPicker = !!getThemeProperty(
+      WEBCHAT.CUSTOM_PROPERTIES.customEmojiPicker,
+      props.enableEmojiPicker
+    )
+    return (
+      getThemeProperty(
+        WEBCHAT.CUSTOM_PROPERTIES.enableEmojiPicker,
+        props.enableEmojiPicker
+      ) ?? hasCustomEmojiPicker
+    )
+  }
+  const emojiPickerEnabled = isEmojiPickerEnabled()
+
+  const isAttacmentsEnabled = () => {
```

Comment:
Typo: Attacments -> Attachments. There are some of them.

Reply:
done